tool/builder3dtools/src/main/java/org/openscience/cdk/modeling/builder3d/TemplateExtractor.java

```diff
@@ -365,8 +365,9 @@ public void makeCanonicalSmileFromRingSystems(String dataFileIn, String dataFile
     public List<IBitFingerprint> makeFingerprintsFromSdf(boolean anyAtom, boolean anyAtomAnyBond,
             Map<String, Integer> timings, BufferedReader fin, int limit) throws Exception {
 
-        IFingerprinter fingerPrinter = new HybridizationFingerprinter(HybridizationFingerprinter.DEFAULT_SIZE,
+        HybridizationFingerprinter fingerPrinter = new HybridizationFingerprinter(HybridizationFingerprinter.DEFAULT_SIZE,
```

Comment:
The basic idea of this one is that sp2 hybridization is a lot more stable than aromaticity...